Sources/Classes/core/SecureStorage/Database/MiniAppSecureStorageSqliteDatabase+Model.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import SQLite
+
+extension MiniAppSecureStorageSqliteDatabase {
+
+    struct Entry: Codable {
+
+        var key: String
+        var value: String
+
+        // MARK: - Database
+        static let entries = Table("entries")
+        static let key = Expression<String>("key")
+        static let value = Expression<String>("value")
+
+        static func migrate(database: Connection) throws {
+            try database.run(entries.create { table in
+                table.column(key, primaryKey: true)
+                table.column(value)
+            })
+        }
+
+        static func find(database: Connection, key: String) throws -> Entry? {
+            for entry in try database.prepare(entries.filter(Self.key == key)) {
+                return Entry(key: entry[Self.key], value: entry[Self.value])
+            }
+            return nil
+        }
+
+        static func upsert(database: Connection, key: String, value: String) throws {
+            let upsert = entries.upsert(Self.key <- key, Self.value <- value, onConflictOf: Self.key)
+            _ = try database.run(upsert)
```

Comment:
can we return the results of upsert/delete etc. and log when something went wrong? would be cleaner in my opinion

Reply:
done 👍